install/kubernetes/cilium/values.yaml

```
@@ -2054,3 +2054,23 @@ enableK8sTerminatingEndpoint: true
 
 # -- Configure whether to unload DNS policy rules on graceful shutdown
 # dnsPolicyUnloadOnShutdown: false
+
+dnsProxy:
+  # -- DNS response code for rejecting DNS requests, available options are '[nameError refused]' (default "refused")
```

No need to document the default value in the comment, it is visible one line below and there is a dedicated column in the generated README.

```suggestion
  # -- DNS response code for rejecting DNS requests, available options are '[nameError refused]'.
```

Ditto for the other values introduced.